PixyViSy.cpp

```diff
@@ -15,48 +15,77 @@ void PixyViSy::update(void)
   uint8_t all_blocks_count = pixy.getBlocks();
   left_pixels = 0;
   right_pixels = 0;
-  action = 'N';
+  action = 'N'; // as "Nothing"
   distance = ~0;
   uint16_t goal_pix_height = 0;
+  uint8_t max[3] = { all_blocks_count, all_blocks_count, all_blocks_count };
+
   if (all_blocks_count) {
+    /* finding three max blocks */
     for (uint8_t i = 0; i < all_blocks_count; i++) {
-      if (pixy.blocks[i].signature == signature) {
+      if (pixy.blocks[i].signature == signature && BLOCK_SIZE(pixy.blocks[i]) >= MIN_BLOCK_AREA) {
+        if (max[2] == all_blocks_count || BLOCK_SIZE(pixy.blocks[i]) > BLOCK_SIZE(pixy.blocks[max[2]])) {
+          if (max[1] == all_blocks_count || BLOCK_SIZE(pixy.blocks[i]) > BLOCK_SIZE(pixy.blocks[max[1]])) {
+            if (max[0] == all_blocks_count || BLOCK_SIZE(pixy.blocks[i]) > BLOCK_SIZE(pixy.blocks[max[0]])) {
+              max[2] = max[1];
+              max[1] = max[0];
+              max[0] = i;
+            }
+            else {
+              max[2] = max[1];
+              max[0] = i;
+            }
+          }
+          else {
```

Comment:
@vargac I think we should decide on which way of formatting will be used in this project. I personally like the other way (having `}` on the same line as `else {`) a bit more.